s3watcher/lambda/lib/Logger.js

```diff
@@ -35,15 +34,11 @@ module.exports = {
     messages: messages,
 
     log: function (stage, messageKey, state) {
-        log(stage, messageKey, state)
+        log(stage, messageKey, state);
```

Comment:
its not in the [ci script](https://github.com/guardian/grid/blob/master/s3watcher/build.sh)... oops! will raise a PR to fix.